weblate/addons/models.py

```diff
@@ -208,34 +209,36 @@ def handle_addon_error(addon, component):
         addon.disable()
 
 
+def handle_addon_event(sender, component, translation, event_type, store):
```

Comment:
How about avoiding using confusing names and pass attributes to the method directly?

```suggestion
def handle_addon_event(sender, component, event_type, **kwargs):
```

Reply:
Done